include/mbgl/util/indexed_tuple.hpp

```diff
@@ -30,18 +30,18 @@ class IndexedTuple<TypeList<Is...>, TypeList<Ts...>> : public std::tuple<Ts...>
     static_assert(sizeof...(Is) == sizeof...(Ts), "IndexedTuple size mismatch");
 
     template <class I>
-    auto& get() {
+    auto& get() noexcept {
         return std::get<TypeIndex<I, Is...>::value>(*this);
     }
 
     template <class I>
-    const auto& get() const {
+    const auto& get() const noexcept {
         return std::get<TypeIndex<I, Is...>::value>(*this);
     }
 
     template <class... Us>
-    IndexedTuple(Us&&... other)
-        : std::tuple<Ts...>(std::forward<Us>(other)...){};
+    IndexedTuple(Us&&... other) noexcept(std::is_nothrow_constructible_v<std::tuple<Ts...>, Us&&...>)
```

Comment:
Pretty sure this is always false.

```suggestion
    IndexedTuple(Us&&... other)
```